functions/src/index.js

```diff
@@ -1,133 +1,207 @@
-import shuffle from 'lodash/shuffle';
-import sortBy from 'lodash/sortBy';
-import { firestore } from 'firebase-functions';
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import { shuffle } from 'lodash';
 
+admin.initializeApp(functions.config().firebase);
+
+/**
+ *
+ * @param cards
+ * @param player
+ * @returns {{cards: *}}
+ */
 const assignCardsToPlayer = (cards, player) => ({ ...player, cards });
 
-function onDealCards(event) {
-    const { players } = event.data.data();
-    const cards = shuffle([
-        '7S',
-        '8S',
-        '9S',
-        '10S',
-        'AS',
-        'JS',
-        'QS',
-        'KS',
-        '7H',
-        '8H',
-        '9H',
-        '10H',
-        'AH',
-        'JH',
-        'QH',
-        'KH',
-        '7D',
-        '8D',
-        '9D',
-        '10D',
-        'AD',
-        'JD',
-        'QD',
-        'KD',
-        '7C',
-        '8C',
-        '9C',
-        '10C',
-        'AC',
-        'JC',
-        'QC',
-        'KC',
-    ]);
+/**
+ *
+ * @type {any | Array}
+ */
+const cards = [
+    '7S',
+    '8S',
+    '9S',
+    '10S',
+    'AS',
+    'JS',
+    'QS',
+    'KS',
+    '7H',
+    '8H',
+    '9H',
+    '10H',
+    'AH',
+    'JH',
+    'QH',
+    'KH',
+    '7D',
+    '8D',
+    '9D',
+    '10D',
+    'AD',
+    'JD',
+    'QD',
+    'KD',
+    '7C',
+    '8C',
+    '9C',
+    '10C',
+    'AC',
+    'JC',
+    'QC',
+    'KC',
+];
 
+/**
+ *
+ * @param players
+ * @returns {*}
+ */
+function dealCards(players) {
```

Comment:
can we use split this to a different "service" file that has no side effects and not related to functions?